keyboards/tiger910/keymaps/rev1/keymap.c

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Not sure this keymap adds value over default, so should be removed.

Reply:
This version changes the number of RGB lights, so you need to modify the RGB control statement.